sys/fmt/fmt.c

```diff
@@ -118,6 +123,25 @@ size_t fmt_s32_dec(char *out, int32_t val)
     return fmt_u32_dec(out, val) + negative;
 }
 
+size_t fmt_s16_dec(char *out, int16_t val)
```

Comment:
AFAIK the compiler will inline it automatically anyway, same goes for `fmt_u16_dec`...